src/lib/shared_utils/src/canister_specific/individual_user_template/types/hot_or_not/mod.rs

```diff
@@ -395,18 +395,21 @@ impl Post {
                 let denominator = DURATION_OF_EACH_SLOT_IN_SECONDS;
                 let currently_ongoing_slot = ((numerator / denominator) + 1) as u8;
 
-                let temp_room_details_default = RoomDetailsV1::default();
+                // let temp_room_details_default = RoomDetailsV1::default();
 
                 // get currently active room
                 let active_room_id = slot_details_map
                     .get(&(self.id, currently_ongoing_slot))
-                    .unwrap_or(SlotDetailsV1::default())
+                    .unwrap_or_default()
```

Comment:
this is a minor change. just for sanity.

Reply:
just to understand, the functionality remains same but more idiomatic right ?

Reply:
yes. you are right.